src/types/config.js

```diff
@@ -25,7 +25,11 @@ export type ConfigMenuGroup = {
 };
 
 export type Config = {
+  archiveEnabled: ?boolean,
   dependencies?: { dagMaxServicesLen?: number, menuEnabled?: boolean },
-  gaTrackingID?: ?string,
+  tracking?: {
+    gaID: ?string,
+    trackErrors: ?boolean,
+  },
```

Comment:
This changes to add `tracking` to config is something I missed when adding the GA error tracking.